src/NHSD.BuyingCatalogue.Database/Tables/CataloguePrice.sql

```diff
@@ -1,14 +1,16 @@
 ﻿CREATE TABLE dbo.CataloguePrice
 (
     CataloguePriceId int IDENTITY(1,1) NOT NULL,
-    CatalogueItemId nvarchar(14) NOT NULL,
+    CatalogueItemId varchar(14) NOT NULL,
     ProvisioningTypeId int NOT NULL,
     CataloguePriceTypeId int NOT NULL,
     PricingUnitId uniqueidentifier NOT NULL,
     TimeUnitId int NULL,
-    CurrencyCode nvarchar(3) NOT NULL,
+    CurrencyCode varchar(3) NOT NULL,
     LastUpdated datetime2(7) NOT NULL,
+    Price decimal(18,3) NULL,
     CONSTRAINT PK_CataloguePrice PRIMARY KEY CLUSTERED (CataloguePriceId),
```

Comment:
Do we want to omit the `CLUSTERED` keyword until we decide on the preferred convention so that it is consistent across the project?

Reply:
Yes we should definitely try to be consistent where possible. Removed the keyword.